mediaflow_proxy/routes/playlist_builder.py

```
@@ -0,0 +1,384 @@
+            elif 'vavoo.to' in logical_line:
+                encoded_url = urllib.parse.quote(logical_line, safe='')
+                processed_url_content = f"{base_url}/proxy/hls/manifest.m3u8?d={encoded_url}&api_password={api_password}"
+            elif 'vixsrc.to' in logical_line:
+                encoded_url = urllib.parse.quote(logical_line, safe='')
+                processed_url_content = f"{base_url}/extractor/video?host=VixCloud&redirect_stream=true&d={encoded_url}&api_password={api_password}"
+            elif '.m3u8' in logical_line:
+                encoded_url = urllib.parse.quote(logical_line, safe='')
+                processed_url_content = f"{base_url}/proxy/hls/manifest.m3u8?d={encoded_url}&api_password={api_password}"
+            elif '.mpd' in logical_line:
+                # Estrai parametri DRM dall'URL MPD se presenti
+                import re
```

Ruff (0.12.2)

`re` imported but unused

Remove unused import: `re`

(F401)